test/utils/azure/azure_helpers.go

```diff
@@ -299,7 +314,16 @@ func getClient(env azure.Environment, subscriptionID, tenantID string, armSpt *a
 		sshPublicKeysClient: compute.NewSSHPublicKeysClientWithBaseURI(env.ResourceManagerEndpoint, subscriptionID),
 	}
 
-	authorizer := autorest.NewBearerAuthorizer(armSpt)
+	if !strings.HasSuffix(scope, "/.default") {
+		scope += "/.default"
+	}
+	// Use an adapter so azidentity in the Azure SDK can be used as Authorizer
+	// when calling the Azure Management Packages, which we currently use. Once
+	// the Azure SDK clients (found in /sdk) move to stable, we can update our
+	// clients and they will be able to use the creds directly without the
+	// authorizer.
+	authorizer := azidext.NewTokenCredentialAdapter(cred, []string{scope})
```

Comment:
@MartinForReal do you know how to fix this now?

Reply:
As mentioned in comments, we need to migrate our model from the ones in /service to the ones in /sdk/resourcemanager

Reply:
not an issue now since this folder is testing code.